<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QListWidget" name="list_files">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btn_file_open">
        <property name="text">
         <string>File Open...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_extract">
        <property name="text">
         <string>Extract</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>check &quot;tr&quot;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_context">
        <property name="styleSheet">
         <string notr="true"> QGroupBox {
	border: 1px solid rgb(196, 202, 209);
	border-radius: 5px;
	margin-top: 10px;
	margin-bottom: 7px;
	padding:0px;
	padding-top:1ex;
 }

 QGroupBox::title {
	top: -10 ex;
	left: 10px;
	subcontrol-origin: border;
    padding: 2px;
color: rgb(79, 84, 92);
 }
</string>
        </property>
        <property name="title">
         <string>Context Type</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QRadioButton" name="radioButton_fixed">
           <property name="text">
            <string>fixed context</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_section">
           <property name="text">
            <string>section context</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_file">
           <property name="text">
            <string>filename context</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_change_context">
           <property name="text">
            <string>Change Context</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_open_work_directory">
        <property name="text">
         <string>Work Directory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_exit">
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
